### ISSUE TITLE
Change signature of MultiResNDTiffAPI.increaseMaxResolution

### ISSUE THREAD

**Issue by nicost** [micro-manager/NDTiffStorage#141]
Is it OK to change the signature of MultiResNDTiffAPI 

`void increaseMaxResolutionLevel(int newMaxResolutionLevel);`

to:

`Future<Integer> increaseMaxResolutionLevel(int newMaxResolutionLevel);`

where Integer denotes the newMaxResolutionIndex?

This would make it possible to immediately display the new resolution image, rather than having to wait for a user action to update the UI.  Currently, in Magellan in Explore mode, zooming out regularly leads to a completely black canvas that only gets filled in when the user moves the mouse. or otherwise forces the display to update.

**Comment by henrypinkard** [comment]
Yes, and that makes sense. Here are all the places where it is used, which (maybe?) you would need to update. Though maybe not because they can just discard the future it returns 


![Screen Shot 2023-11-02 at 8 47 46 AM](https://github.com/micro-manager/NDTiffStorage/assets/7969470/8d8fdf6d-ac03-478e-8d13-1c9b7b9dc8ab)
